packages/graphql-hooks/index.d.ts

```diff
@@ -50,6 +50,19 @@ export class GraphQLClient {
   ): Promise<Result<ResponseData, TGraphQLError>>
 }
 
+export class LocalGraphQLClient extends GraphQLClient {
+  constructor(options: LocalClientOptions)
+}
+interface LocalClientOptions extends ClientOptions {
+  localQueries: LocalQueries
```

Comment:
This one is awkward, because the parent class has `url` required the types would conflict. One problem of using inheritance here. I haven't really found any easy solution other than keeping it as it is (and it means people would have to pass a placeholder `url` value if using typescript).

This could be solved if we changed the implementation to having `GraphQLConnection` objects instead of inheriting the client, or found some other solution, but I figured the drawback of having to specify this one property isn't too bad until we can improve it